nodejs/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# Install specified version of node.js into the system
+
+# Add nodesource to RPM. Unfortunately, the URL is version dependent. When we
+# upgrade node.js, we will have to upgrade this URL as well.
+- name: add node.js rpm to yum
+  shell: 'curl --silent --location https://rpm.nodesource.com/setup_4.x | bash -'
+
+- name: install nodejs
+  yum:
+    name: "nodejs-{{ nodejs_version }}"
+
+# Disable yum from updating node.js since we are locking down this version for
+# the application.
+- name: disable nodejs from yum repo
```

Comment:
nice catch 👍 

Hopefully travis updates (or we move away from it) and then we can upgrade node